src/main/scala/DocGenerator.scala

```diff
@@ -56,7 +60,7 @@ object LatexDocGenerator extends DocGenerator {
       case p: Pkg =>
         p +: loop(p)
     }.flatten
-    loop(root)
+    loop(root) :+ root
   }
```

Comment:
I think an extractor method like this should go into a separate place where it can be re-used by multiple backends